templates/quoref/templates.yaml

```diff
@@ -0,0 +1,70 @@
+dataset: quoref
+templates:
+  6f1d5031-1377-4b8a-9475-987b2275b8da: !Template
+    id: 6f1d5031-1377-4b8a-9475-987b2275b8da
+    jinja: 'Given the following, context:
+
+      {{context}}
+
+      ===
```

Comment:
Same comment from your other PR about ===.

Reply:
Sure, I will update the prompts.

Reply:
Updated! @craffel